tests/zexperimental/test_mcmc.py

```diff
@@ -115,21 +115,20 @@ def test_MH_regression_warmup(target: cuqi.density.Density):
     sampler_new = cuqi.experimental.mcmc.MHNew(target, scale=1)
     assert_true_if_warmup_is_equivalent(sampler_old, sampler_new)
 
-# ============ pCN ============
+# ============ PCN ============
 
 @pytest.mark.parametrize("target", targets)
 def test_pCN_regression_sample(target: cuqi.density.Density):
     """Test the pCN sampler regression."""
     sampler_old = cuqi.sampler.pCN(target, scale=1)
-    sampler_new = cuqi.experimental.mcmc.pCNNew(target, scale=1)
+    sampler_new = cuqi.experimental.mcmc.PCNNew(target, scale=1)
     assert_true_if_sampling_is_equivalent(sampler_old, sampler_new)
 
 @pytest.mark.parametrize("target", targets)
-@pytest.mark.xfail(reason="The warmup is not equivalent at this point for pCN sampler.")
 def test_pCN_regression_warmup(target: cuqi.density.Density):
     """Test the pCN sampler regression."""
     sampler_old = cuqi.sampler.pCN(target, scale=1)
-    sampler_new = cuqi.experimental.mcmc.pCNNew(target, scale=1)
+    sampler_new = cuqi.experimental.mcmc.PCNNew(target, scale=1)
```

Comment:
Just to be sure, can we try with a different scale e.g. 0.001?

Reply:
Done